src/Edelstein.Service.Game/Commands/Handling/LevelUpCommand.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Threading.Tasks;
+using CommandLine;
+using Edelstein.Service.Game.Fields.Objects.User;
+
+namespace Edelstein.Service.Game.Commands.Handling
+{
+    public class LevelUpCommand : AbstractCommand<LevelUpCommandOption>
+    {
+        public override string Name => "LevelUp";
+        public override string Description => "Undergoes leveling up process";
+
+        public LevelUpCommand(Parser parser) : base(parser)
+        {
+            Aliases.Add("LvlUp");
+            Aliases.Add("LVLUP");
+        }
+
+        protected override async Task Execute(FieldUser sender, LevelUpCommandOption option)
+        {
+            if (option.Value < 0)
+            {
+                await sender.Message($"ERROR: this command cannot de-level character use only positive integers!");
```

Comment:
To keep it consistent with the other message right now I think there is no need for the 'ERROR: ' prefix.